packages/web3.js-plugin-fee-estimation/package.json

```
@@ -0,0 +1,45 @@
+{
+  "name": "@eth-optimism/web3.js-plugin-fee-estimation",
+  "version": "0.1.0",
+  "description": "A Web3.js plugin for doing OP-Chain gas estimation",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ethereum-optimism/optimism.git",
+    "directory": "packages/web3js-plugin-fee-estimation"
+  },
+  "homepage": "https://optimism.io",
+  "type": "module",
+  "main": "dist/plugin.js",
+  "module": "dist/plugin.mjs",
+  "types": "src/plugin.ts",
+  "files": [
+    "dist/",
+    "src/"
+  ],
+  "scripts": {
+    "build": "tsup",
+    "lint": "prettier --check .",
+    "lint:fix": "prettier --write .",
+    "test": "vitest",
```

```suggestion
    "test": "vitest --coverage",
```
Not opinionated here but I started recently doing this. Up to you if you prefer to not show or show coverage